package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@snapshot-labs/snapshot.js",
-  "version": "0.1.0",
-  "repository": "snapshot-labs/snapshot.js",
+  "name": "@vfatouros/snapshot.js",
```

Comment:
Can you remove the changes here, on package-lock and on test/index.ts? Thanks!

Reply:
Sorry about those! With the test file I wasn't sure if you run the new strategy test too before deploying